ts/store/reducers/wallet/wallets.ts

```diff
@@ -209,21 +209,7 @@ export const getPayablePaymentMethodsSelector = createSelector(
     potPm: ReturnType<typeof paymentMethodsSelector>
   ): ReadonlyArray<PaymentMethod> =>
     pot.getOrElse(
-      pot.map(potPm, pms => pms.filter(canMethodPay)),
-      []
-    )
-);
-
-// return those payment methods that have pagoPA as enabled function
-export const getPagoPAMethodsSelector = createSelector(
-  paymentMethodsSelector,
-  (
-    potPm: ReturnType<typeof paymentMethodsSelector>
-  ): ReadonlyArray<PaymentMethod> =>
-    pot.getOrElse(
-      pot.map(potPm, pms =>
-        pms.filter(pm => hasFunctionEnabled(pm, EnableableFunctionsEnum.pagoPA))
-      ),
+      pot.map(potPm, pms => pms.filter(hasPaymentFeature)),
```

Comment:
Since the rename to `hasPaymentFeature`, could be worth renaming also the `getPayablePaymentMethodsSelector` in order to have the same name(es. `withPaymentFeatureSelector`)?

Reply:
Sounds good! This is +1 to defeat misleading naming of functions and data in the payment world 💪🏻 
changed here ab9e1db72a9bfc7547c1f8bd62969b63ec48609c